pkg/ingester/pyroscope/ingest_adapter.go

```diff
@@ -141,7 +151,7 @@ func convertMetadata(pi *storage.PutInput) (metricName, stType, stUnit, app stri
 		stType = stTypeSamples
 		stUnit = stUnitCount
 	case "wall":
```

Comment:
This is kind of separate, but @korniltsev what do you think about defaulting to CPU if there's no prefix instead of throwing an error? (`default` case)

Reply:
I don't have a strong opinion on this matter, I'm not opposed to this solution, I just thought that personally, I would notice an error in the logs faster than incorrect data in the UI.